R/exclusions.R

```diff
@@ -13,7 +13,7 @@ exclude_from_title <- function(data) {
     dplyr::arrange(dplyr::desc(publication_date)) |>
     dplyr::distinct(title, .keep_all = TRUE) |>
     # Items related to surgery
-    exclude(title, "surgery|surgical|angle glaucoma|fracture|abdomen|abdominal|wound|vein|resection|urethroplasty|ectomy|inguinal|aortic|rhinoplasty|lumbar") |>
+    exclude(title, "surgery|surgical|angle glaucoma|fracture|abdomen|abdominal|wound|vein|resection|urethroplasty|ectomy|inguinal|aortic|rhinoplasty|lumbar|health|clinical|disease|patient|medical") |>
```

Comment:
I think on their own, these exclusions might drop a lot more than we expect. I'll look into it though

Reply:
Definitely!!